src/Frontend/Page/Comment.hs

```diff
@@ -29,11 +29,10 @@ import LifeCycle
 import qualified Frontend.Path as U
 
 data CommentWidget = CommentWidget
-    { _cwIdeaCaps :: [IdeaCapability]
-    , _cwUserId   :: AUID User
-    , _cwRole     :: Role
-    , _cwContext  :: CommentContext
-    , _cwComment  :: Comment
+    { _cwIdeaCaps      :: [IdeaCapability]
```

Comment:
There is a type synonym for `[IdeaCapability]` that is `IdeaCapabilities` please use that, as the same style is applied on the persistent layer.